type.c

```diff
@@ -747,11 +747,11 @@ int FC_FUNC( mpi_type_ub, MPI_TYPE_UB )(int * type, MPI_Aint * ub, int * ierr)
   return MPI_SUCCESS;
 }
 
-int MPI_Type_ub(MPI_Datatype type, MPI_Aint * ub)
+int MPI_Type_ub(MPI_Datatype type, MPI_Aint *displacement)
```

Comment:
Some of these changes are just changes in variable names. Do we need it?

Reply:
I just want to make them exactly the same as MPICH in this PR, even though this change is not strictly necessary.